live-site/components/hacking-remaning/HackingRemaining.styles.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledHackingRemainingContainer = styled.div`
+  z-index: 20;
+  float: right;
+  background-color: ${colors.BUTTON_DARK_GREEN};
+
+  border-radius: 1em;
```

Comment:
The right side shouldn't have a border radius so let's only apply border radius to the left corners

<img width="233" alt="Screenshot 2022-11-23 at 11 02 45 AM" src="https://user-images.githubusercontent.com/59738880/203592906-8d985dd4-eb75-4daf-835b-1a9de88ac0f1.png">
<img width="231" alt="Screenshot 2022-11-23 at 11 02 52 AM" src="https://user-images.githubusercontent.com/59738880/203592937-c2cb873a-e6b8-4c61-806d-329aaf08d061.png">